### ISSUE TITLE
Replace Google Analytics with Mixpanel

### ISSUE THREAD

**Issue by jviotti** [resin-io/etcher-homepage#119]

**Comment by izavits** [comment]
There's already a mixpanel project gathering events from that project. (if I understand correctly this is the etcher.io website)

**Comment by jviotti** [comment]
@izavits Correct, but I believe Google Analytics is still being setup in this app, so lets make sure the Mixpanel integrations contains everything thing that we need, and lets completely kill GA
